server/common/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+
+server:
+  verbose: true
+  debug: false
+  host: "127.0.0.1"
+  port : null
+  scripts : []
+  open_browser: false
+
+multi_dataset:
+  dataroot: null
+
+  # The index page when in multi-dataset mode:
+  #   false or null:  this returns a 404 code
+  #   true:  loads a test index page, which links to the datasets that are available in the dataroot
+  #   string/URL:  redirect to this URL:  flask.redirect(config.multi_dataset__index)
+  index: false
+
+  # A list of allowed matrix types.  If an empty list, then all matrix types are allowed
+  allowed_matrix_types: []
+
+single_dataset:
+  datapath: null
+  obs_names: null
+  var_names: null
+  about: null
+  title: null
+
+annotations:
+  enable: true
+  type: local_file_csv
+  local_file_csv:
+    directory: null
+    file: null
+  ontology:
+    enable: false
+    obo_location: null
+  max_categories: 1000
```

Comment:
not sure that `max_categories` belongs in this group.  The parent `annotations` is really controlling user-created annotations. The `max_categories` affects all annotations, even those in the base data (immutable), and is actually affecting the presentation of the annotations, not the ability to create/edit them.

Suggestions:
1. change top-level `annotations` to `user_anotations`
2. create a new top-level category `presentation`, and move `max_categories` into it

We have some new features (eg, color palette) that seem likely to need the `presentation` group.

Reply:
Sounds good.   I'll make that change